h2o-py/h2o/model/model_base.py

```diff
@@ -129,7 +129,7 @@ def predict(self, test_data):
         if not isinstance(test_data, h2o.H2OFrame): raise ValueError("test_data must be an instance of H2OFrame")
         j = H2OJob(h2o.api("POST /4/Predictions/models/%s/frames/%s" % (self.model_id, test_data.frame_id)),
                    self._model_json["algo"] + " prediction")
-        j.poll()
+        j.poll()        # warning messages associated with job now
```

Comment:
Before my fix, the warning messages are not associated with jobs. Now it is.  However, it is just FYI for me, nobody else need to know. Will remove it.